src/material/card/testing/card-harness.e2e.spec.ts

```diff
@@ -6,7 +6,7 @@ import {browser} from 'protractor';
 describe('card harness', () => {
   let loader: HarnessLoader;
 
-  beforeEach(async () => await browser.get('/mdc-card'));
+  beforeEach(async () => await browser.get('/cards'));
```

Comment:
nit: shouldn't this be `card`?

Reply:
For some reason no, its /cards in the e2e app